db/migrate/20211023154550_add_hide_from_all_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddHideFromAllToUser < ActiveRecord::Migration[5.2]
```

Comment:
```suggestion
class AddHideFromAllToUser < ActiveRecord::Migration[7.1]
```
Wow, we added this a long time ago :eyes: 7.1 should be a valid Migration config here